tests/test_models/test_necks/test_necks.py

```diff
@@ -56,3 +56,26 @@ def test_imvoxel_neck():
     inputs = torch.rand([1, 64, 216, 248, 12], device='cuda')
     outputs = neck(inputs)
     assert outputs[0].shape == (1, 256, 248, 216)
+
+
+def test_dla_neck():
+    if not torch.cuda.is_available():
```

Comment:
Same question here. Is GPU necessary? If not, we can make this unit test run on CPU.

Reply:
In dla_neck, because of DeformConv, GPU is necessary.

Reply:
I see.